### ISSUE TITLE
Cloning Grofers.com for the platfrom

### ISSUE THREAD

**Issue by reeversedev** [Grommerce/grommerce-store#5]
**Is your feature request related to a problem? Please describe.**
We have a major problem of designs and that is something which is stopping us from building something useful so why not make a replica of some website? After all, it's for Non Profit uses.
**Describe the solution you'd like**
Clone https://grofers.com/ because they have decent designs and can help us building things on our own.
**Describe alternatives you've considered**
I would have recommended Big Basket as well but I didn't like their design that much.

**Comment by bhautikbharadava** [comment]
Agreed I looked both sites and they are likely same apart from some things. I think we should go for grofers. Let's start with it and after that will look for changes

**Comment by reeversedev** [comment]
Yeah sure. I'll start with creating few issues today.
